### ISSUE TITLE
App doesn't start 

### ISSUE THREAD

**Issue by danielthelegendary** [Akylas/oss-weather#270]
Xiaomi redmi Note 9 pro android 13.0.2 no root, using the newest app version, guaranteed.

**Comment by farfromrefug** [comment]
@danielthelegendary was it starting before? Where did you install the app from?

**Comment by danielthelegendary** [comment]
F-Droid Izzy Repo. Here is the log. I just installed an older version which worked. ![Screenshot_2024-12-14-00-29-59-947_com.miui.bugreport.jpg](https://github.com/user-attachments/assets/2c401f43-e149-4805-a87c-a810fae7d631)

**Comment by farfromrefug** [comment]
@danielthelegendary awesome thanks. Are you really sure you have the latest version? almost certain i fixed that already

**Comment by danielthelegendary** [comment]
Izzy provides me with 2.9.3.146 

Maybe there is a time lag in the repo.

**Comment by danielthelegendary** [comment]
Yeah let's do that. Thanks for the awesome support! Mail address?

**Comment by farfromrefug** [comment]
@danielthelegendary use contact at akylas dot fr

**Comment by 0-5-0** [comment]
@danielthelegendary I'm curious if you're having the same crash on the latest version (build 147 for android), if it was related to a chart bug that got fixed.

**Comment by 0-5-0** [comment]
This can be closed?
